packages/tenable_sc/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.4.1"
+  changes:
+    - description: Add support for endless pagination.
+      type: enhancement
```

This sounds like some magic new feature. 😄  But shouldn't this be a `bugfix`? The info in the PR description would be good here.


```suggestion
    - description: Fix an indefinite pagination bug by adding explicit pagination termination conditions. In Agent versions >= 8.2.0 pagination termination was never happening.
      type: bugfix
```